src/com/sbezboro/standardplugin/commands/EndresetCommand.java

```diff
@@ -22,7 +23,16 @@ public boolean handle(CommandSender sender, Command command, String label, Strin
 		
 		if (plugin.getEndResetManager().isEndResetScheduled()) {
 			long endReset = plugin.getEndResetStorage().getNextReset();
-			sender.sendMessage(ChatColor.BLUE + "The next end reset is on " + ChatColor.AQUA + MiscUtil.friendlyTimestamp(endReset));
```

Comment:
can you put the changes related to end resets in another PR? I want to deploy the changes here with just the PVP changes for now. End reset stuff can be disabled after the reset until things are more stable